invoicer_project/static/js/add_edit_item.js

```diff
@@ -0,0 +1,166 @@
+import {nameField, priceField, amountInStockField, barcodeField, additionalFieldsContainer, amountAdditionalFieldsContainer} from './utils_items.js'
+import { removeAllErrorAttributes } from './validation_utils.js';
+
+document.querySelectorAll('.dropdown_list').forEach(function (dropdownWrapper) {
```

Comment:
Why don't we listen to events on parent node? Are you sure we need to add manually event listener to each node?
Let me explain what I mean.

If we have html
```html
<ul>
    <li>one</li>
   <li>two</li>
  <li>three</li>
</ul>
```

```js
document.querySelector('ul').addEventListener('click', e => {
      const target = e.target;
     if(target.nodeName === 'LI') {
        console.log("You clicked on li node');
    }
});
```
I prefer to add callback to parent node because children nodes could be added to parent in any time and in that case you will not need to add listeners manually

Reply:
Done

Reply:
I haven't seen any changes

Reply:
Removed some callbacks, is it what you mean?